src/app/main/component/user/components/profile/edit-profile/edit-profile.component.html

```diff
@@ -20,7 +20,8 @@ <h2>{{ 'user.edit-profile.edit-profile' | translate }}</h2>
           <div class="input-block">
             <label for="city">{{ 'user.edit-profile.city' | translate }}</label>
             <app-input-google-autocomplete
-              formControlName="city"
+              id="city"
+              [control]="editProfileForm.get('city')"
```

Comment:
Why use this syntax if the control is static and doesn't change dynamically?

Reply:
> Why use this syntax if the control is static and doesn't change dynamically?

Yes, you are right, updated `[formControl]="city"` instead
and cast `city` the return value to a `FormControl`